atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -211,7 +211,7 @@ public Set<CassandraServer> getInitialServerList() {
     }
 
     private Set<InetSocketAddress> getServersFromConfig() {
-        return config.servers().accept(new ThriftHostsExtractingVisitor());
+        return runtimeConfig.get().servers().accept(new ThriftHostsExtractingVisitor());
```

Comment:
Bringing attention to this: Given that CassandraService already does some live reloading off servers, and falls back to the config when it failed to resolve the token ring, I'm not doing anything fancy for live reloading here - just using the runtime config value instead of the install config